prestoadmin/prestoclient.py

```diff
@@ -296,3 +301,11 @@ def _get_private_key(self, keystore):
             raise Exception('No alias %s found in %s. Available aliases are %s' %
                             (alias, self.coordinator_config.get_client_keystore_path(),
                              all_keys))
+
+    def _add_auth_headers(self, existing_headers):
+        # base64 encode the username and password
+        if self.coordinator_config.use_ldap():
+            user = self.coordinator_config.get_ldap_user()
```

Comment:
Check that user and password are not empty, and that username does not contain the `:` character.